EnvironmentSpecificSQLFlowClient.java

```diff
@@ -0,0 +1,38 @@
+package org.apache.zeppelin.sqlflow.client.utils;
+
+import java.util.Map;
+import io.grpc.ManagedChannel;
+import io.grpc.ManagedChannelBuilder;
+import org.apache.commons.lang3.StringUtils;
+import org.sqlflow.client.SQLFlow;
+import org.sqlflow.client.MessageHandler;
+import proto.Sqlflow;
+
+public class EnvironmentSpecificSQLFlowClient {
+
+	public static SQLFlow getClient(MessageHandler messageHandler,
+			Map<String, String> parameters) {
+		// Get the configuration parameters from the page side
+		String serverAddr = parameters.get("sqlflow.serverAddr");
+		String submitter = "mlp";
+		String username = parameters.get("mysql.username");
+		String password = parameters.get("mysql.password");
+		String mysqlAddr = parameters.get("mysql.serverAddr");
+		String databaseName = parameters.get("mysql.databaseName");
+		String dataSource = "mysql://" + username + ":" + password + "(" + mysqlAddr
+				+ ")/" + databaseName + "?maxAllowedPacket=0";
+		String userId = "314";
```

Comment:
What is the '314' means?

Reply:
"314" in this case is used to identify users in the background, and it gives a fixed value. Developers can set the value of **userId** as a variable and get the corresponding parameters according to the login user information of Zeppelin. Different login users have different **userId**, so sqlflow can identify which user is using it.

Reply:
So the `userId` should be a variable instead of a fixed one.

Reply:
Yes it is. The value given in this case is a fixed value, which can actually be assigned to `userId` by getting the user name of the zeppelin logged in user.

Reply:
How？